test/functional.js

```diff
@@ -603,5 +603,84 @@ module.exports = {
                 done();
             });
         });
+
+        it('Vue.js is compiled correctly', (done) => {
+            const cwd = process.cwd();
+            after(() => {
+                process.chdir(cwd);
+            });
+
+            const appDir = testSetup.createTestAppDir();
+            /*
+             * Most of the time, we don't explicitly use chdir
+             * in order to set the cwd() to the test directory.
+             * That's because, in theory, you should be able to
+             * run Encore from other directories, give you set
+             * the context. However, in this case, the vue-loader
+             * users load-postcss-config to load the postcss config,
+             * but it uses process.cwd() to find it, instead of the
```

Comment:
context is a webpack thing... and it's meant to be the root from which entry files are loaded. It's unclear to me whether context should also be used for other things, like the directory where postcss.config.js should be loaded (we *do* give it a bit more importance in Encore)